src/components/DateInput/index.js

```diff
@@ -21,6 +21,9 @@ export class DateInput extends React.Component {
             end: e.target.value
         });
     }
+    componentDidMount = () => {
```

Comment:
The native lifecycle methods from React use a different syntax and should not use `arrow functions`. The reason is because whenever you define an arrow function, it creates a reference to `this` and it is bound to the class level method. Because it's not really reusable to the component, we don't need to keep a reference to it.

```
componentDidMount() {
  ...
}
```

Reply:
I replaced the arrow function with regular function syntax